cmd/nomos/status/cluster_state.go

```diff
@@ -55,7 +55,9 @@ func (c *ClusterState) printRows(writer io.Writer) {
 	}
 	for _, repo := range c.repos {
 		fmt.Fprintf(writer, "%s%s\n", util.Indent, util.Separator)
-		repo.printRows(writer)
+		if name == "" || strings.Contains(repo.syncName, name) {
```

Comment:
The --name flag should be an exact match. Can you check the entire string, rather than using `contains`?

Reply:
the logic has been reverted to find an exact string matching